### DISCUSSION TITLE
问下streams和编辑站点里面的Upstream有什么区别？

### DISCUSSION THREAD

**Discussion by zjqwll** [0xJacky/nginx-ui#759] [Q&A]
![1732523891959](https://github.com/user-attachments/assets/94767745-87a5-4624-a5d2-0fa6eff434de)
![1732524005426](https://github.com/user-attachments/assets/41fc2607-1f3c-442e-8692-0e0488028dfd)
问下streams和编辑站点里面的Upstream有什么区别？

**Comment by 0xJacky** [comment]
没什么关联。网站里 upstream 一般是应用层转发 HTTP/HTTPS 等。Stream 管理一般是传输层如 TCP/UDP 的转发。

**Reply by zjqwll** [reply]
> 没什么关联。网站里 upstream 一般是应用层转发 HTTP/HTTPS 等。Stream 管理一般是传输层如 TCP/UDP 的转发。

好的，谢谢老大。
